src/modules/transaction/components/StakeValidate/StakeValidator.js

```diff
@@ -17,43 +19,62 @@ const ItemList = ({ items, heading }) => {
             stake={items[address]}
             title={items[address].name}
             token={token}
+            reward={rewards[address]?.amount}
           />
         ))}
       </div>
     </div>
   );
 };
 
-const InfoColumn = ({ title, children, className }) => (
+const InfoColumn = ({ title, children, className, toolTipText }) => (
   <div className={`${styles.infoColumn} ${className}`}>
-    <span className={styles.infoTitle}>{title}</span>
+    <span className={styles.infoTitle}>
+      {title}
+      {!!toolTipText && (
+        <Tooltip position="top" indent>
+          <p>{toolTipText}</p>
+        </Tooltip>
+      )}
+    </span>
     <span className={styles.infoValue}>{children}</span>
   </div>
 );
 
-const StakeValidator = ({ t, account, summaryInfo }) => {
+const StakeValidator = ({ t, account, summaryInfo, formProps }) => {
+  const { token } = usePosToken();
   const addedLength = Object.keys(summaryInfo.added).length;
   const editedLength = Object.keys(summaryInfo.edited).length;
   const removedLength = Object.keys(summaryInfo.removed).length;
   const sentStakes = account?.pos?.sentStakes?.length ?? 0;
+  const rewards = formProps.rewards;
 
   return (
     <>
-      {addedLength ? <ItemList heading={t('Added stakes')} items={summaryInfo.added} /> : null}
-      {editedLength ? <ItemList heading={t('Changed stakes')} items={summaryInfo.edited} /> : null}
-      {removedLength ? (
-        <ItemList heading={t('Removed stakes')} items={summaryInfo.removed} />
-      ) : null}
+      {!!addedLength && (
+        <ItemList rewards={rewards} heading={t('Added stakes')} items={summaryInfo.added} />
+      )}
+      {!!editedLength && (
+        <ItemList rewards={rewards} heading={t('Changed stakes')} items={summaryInfo.edited} />
+      )}
+      {!!removedLength && (
+        <ItemList rewards={rewards} heading={t('Removed stakes')} items={summaryInfo.removed} />
+      )}
       <div className={styles.infoContainer}>
         <InfoColumn title={t('Total stakes after confirmation')}>
           {`${sentStakes + addedLength - removedLength}/10`}
         </InfoColumn>
-      </div>
-      {/* <div className={styles.txFeeContainer}>
-        <InfoColumn title={t('Transaction Fees')} className="stake-fees">
-          {transaction?.composedFees.Transaction}
+        <InfoColumn
+          title={t('Total rewards')}
+          toolTipText={t(
+            'Total rewards credited to your account in effective to changes in stakes.'
+          )}
+        >
+          <span className={styles.totalReward}>
+            <TokenAmount val={rewards.total.toString()} token={token} />
```

Comment:
![Screenshot 2023-06-29 at 3 51 07 PM](https://github.com/LiskHQ/lisk-desktop/assets/6449871/c958452d-8346-4b9c-b46a-1f251ff0e4ae)
It can happen that sometimes there is no rewards to claim? so when `Total rewards` is `0` amount we can hide it to not confuse user?